relay-server/src/actors/processor.rs

```diff
@@ -2898,8 +2901,7 @@ impl EnvelopeProcessorService {
         use relay_quotas::ItemScoping;
 
         let RateLimitFlushBuckets {
-            mut bucket_limiter,
-            partition_key,
```

Comment:
we need the partition_key to send the SendMetrics msg, but now handle_rate_limiting isn't sending metrics to the EnvelopeManager any longer, but rather to the aggregator